cedar-policy-validator/src/lib.rs

```diff
@@ -16,7 +16,20 @@
 
 //! Validator for Cedar policies
 #![forbid(unsafe_code)]
+#![warn(rust_2018_idioms, clippy::pedantic, clippy::nursery)]
+#![deny(
+    missing_docs,
```

Comment:
Is this worth the effort for internal crates?

Reply:
Maybe not in retrospect, but now that this PR added the doc comments to bring us into compliance, it's probably worth to maintain?

Reply:
Yeah, fine to keep around